travis/cfg/travis_run_pylint_beta.cfg

```diff
@@ -2,16 +2,21 @@
 #  pylint configuration file will display them without affecting your build status.
 
 # Beta message and code:
+#   api-one-deprecated - W8104
```

Comment:
You have a point about < 8.0.
But api.one usage in 8.0 is legal and should not raise an error.